src/main/java/org/pac4j/j2e/filter/RequiresAuthenticationFilter.java

```diff
@@ -81,52 +75,44 @@ public void init(final FilterConfig filterConfig) throws ServletException {
             final Config config = ConfigBuilder.build(configFactoryParam);
             ConfigSingleton.setConfig(config);
         }
-
         this.clientName = getStringParam(filterConfig, Pac4jConstants.CLIENT_NAME, this.clientName);
-        CommonHelper.assertNotNull(Pac4jConstants.CLIENT_NAME, this.clientName);
-
-        this.requireAnyRole = getStringParam(filterConfig, Pac4jConstants.REQUIRE_ANY_ROLE, this.requireAnyRole);
```

Comment:
`requireAnyRole` and `requireAllRoles` handled specific use cases, replaced by authorizers.